sphinxcontrib/confluencebuilder/common.py

```diff
@@ -75,3 +76,13 @@ def verbose(*args, **kwargs):
     def warn(*args, **kwargs):
         if ConfluenceLogger.logger:
             ConfluenceLogger.logger.warning(*args, **kwargs)
+
+    @staticmethod
+    def trace(title, data):
+        try:
+            with io.open('trace.log', 'a', encoding='utf-8') as file:
+                file.write(u'[%s]\n' % title)
```

Comment:
this could cause issues in py2.7

```
u'%s' % unicode('👉')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xf0 in position 0: ordinal not in range(128)
```

Reply:
that is, assuming title is a byte string. I would encode it first

Reply:
Ah, I don't think I fully tested tracing in 2.7. I'll take a look and cleanup.

Reply:
For this trace method, the title value is an internal-only provided (i.e. not a document title). I wouldn't mind just leaving the method as-is and just assume a provided title is a compatible byte string for the running interpreter.